module/VuFind/src/VuFind/RecordDriver/AbstractBase.php

```diff
@@ -277,11 +281,13 @@ public function getRatingData(?int $userId = null)
      */
     public function getRatingBreakdown(array $groups)
     {
-        return $this->getDbTable('Ratings')->getCountsForResource(
-            $this->getUniqueId(),
-            $this->getSourceIdentifier(),
-            $groups
-        );
+        return $this->getDbServiceManager()
```

Comment:
You could simplify this slightly by just using `$this->getDbService()` here to save a line.